document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/model/document/CompanyReport.java

```diff
@@ -37,6 +38,9 @@ public class CompanyReport {
     @JsonProperty("previous_names")
     private List<PreviousNames> previousNames;
 
+    @JsonProperty("corporate_annotation")
+    private List<CorporateAnnotation> corporateAnnotation;
```

Comment:
Is the use of the singular name `corporateAnnotation` intentional here?

Reply:
Yes, the list in the spec is refered to as a `corporate_annotation` rather than `corporate_annotations` which would make more sense in my opinion, but mainly named it so that it matches the content of the jsonproperty